main.c

```diff
@@ -132,10 +132,10 @@ add_module_rpm_artifacts (Pool           *pool,
                           ModulemdModule *module,
                           Id              sdep)
 {
-  GStrv rpm_artifacts = modulemd_simpleset_dup (modulemd_module_peek_rpm_artifacts (module));
+  g_auto(GStrv) rpm_artifacts = modulemd_simpleset_dup (modulemd_module_peek_rpm_artifacts (module));
   g_auto(Queue) sel;
   queue_init (&sel);
-  for (; *rpm_artifacts; rpm_artifacts++)
+  for (GStrv artifacts = rpm_artifacts; *artifacts; artifacts++)
     {
       const char *nevra = *rpm_artifacts;
```

Comment:
Should be `const char *nevra = *artifact;`

Reply:
Ops, tks. I tested with the correct fix but forgot to include it in the patch.